simulation/sub8_gazebo/src/sub8_thruster_config.cpp

```diff
@@ -35,7 +35,7 @@ try
 catch(XmlRpcException& e)
 {
   auto err_str = "Error parsing port definitions: %s";
```

Comment:
The err_str is not converted to c string, the XMLRpc exception call (which returns std::string) is.